themes/microvideo/template.html

```diff
@@ -401,13 +401,15 @@
 </template>
 
 <media-controller
+  style="--_control-bar-place-self:{{controlbarplace ?? 'unset'}}"
```

Comment:
What is this? Do we need it, or can we comment on it?

Reply:
yep, def needed. microvideo allows placement of the controlbar left, right, top, bottom, center etc.